src/main/java/reposense/parser/RepoConfigCsvParser.java

```diff
@@ -46,9 +46,22 @@ public RepoConfigCsvParser(Path csvFilePath) throws IOException {
     protected void processLine(List<RepoConfiguration> results, String[] elements) throws InvalidLocationException {
         RepoLocation location = new RepoLocation(getValueInElement(elements, LOCATION_POSITION));
         String branch = getValueInElement(elements, BRANCH_POSITION, RepoConfiguration.DEFAULT_BRANCH);
+        boolean isFormatsOverriding = isElementOverridingStandaloneConfig(elements, FILE_FORMATS_POSITION);
+        if (isFormatsOverriding) {
+            removeOverrideKeywordFromElement(elements, FILE_FORMATS_POSITION);
+        }
         List<Format> formats = Format.convertStringsToFormats(getManyValueInElement(elements, FILE_FORMATS_POSITION));
+        boolean isIgnoreGlobListOverriding = isElementOverridingStandaloneConfig(elements, IGNORE_GLOB_LIST_POSITION);
```

Comment:
Let's leave a whitespace between the getting of each value here, make it look neater